src/__tests__/orchestrator.integration.spec.ts

```diff
@@ -165,6 +168,35 @@ describe('elastic orchestrator', () => {
       expect(newIndexDocuments).toEqual(oldIndexDocuments)
     })
 
+    it('will trigger a reindex from the index with the same name for a new index', async () => {
+      const existingConfig = ['index-00001.json', 'another-index-00000.json']
+      await manageIndices(client, existingConfig, indexConfigFileFolder)
+
+      const indices = await getExistingIndices(client)
+      const latestIndex = await getMostRecentIndex(
+        client,
+        indices.filter(index => index.startsWith('another-index-')),
```

Comment:
I guess it doesn't make a difference but we might want to exclude the last -, so we only include what is considered to be the `index name`

```suggestion
        indices.filter(index => index.startsWith('another-index')),
```